src/main/java/pl/ostek/internet_chat/exception/ContactWithYourselfException.java

```diff
@@ -0,0 +1,7 @@
+package pl.ostek.internet_chat.exception;
+
+public class ContactWithYourselfException extends RuntimeException{
```

Comment:
Związane z poprzednim komentem.

Generalnie to zbyt detaliczny już moim zdaniem exception - czy taki exception może wymagać specjalnego obsłużenia?  Może to bardziej ogólny IncorrectContact (np. dla nieistniejących też id-ków), albo w ogóle błąd walidacji ConstraintValidationException